static/src/components/app-root/app-root.tsx

```diff
@@ -61,21 +66,25 @@ export class AppRoot {
         <ion-menu side="start" type="reveal" content-id="main-content" ref={el => this.menu = el}>
           <ion-content>
             <ion-list>
-              <ion-item href="/" lines="none">
+              <ion-item class={{ active: this.pageTitle === 'Home' }} href="/" lines="none">
```

Comment:
Create an array as a class variable, to consolidate the list of pages into one place for creating these menu links.